build/x86_64_sles12/Dockerfile

```diff
@@ -28,9 +28,10 @@ RUN zypper addrepo https://download.opensuse.org/repositories/devel:libraries:c_
         varnish-devel \
         wget \
         which \
- && wget ftp://ftp.pbone.net/mirror/ftp5.gwdg.de/pub/opensuse/repositories/home:/Herbster0815/openSUSE_Factory/noarch/automake-1.14.1-56.1.noarch.rpm \
- && rpm --force -Uvh automake-1.14.1-56.1.noarch.rpm \
- && rm automake-1.14.1-56.1.noarch.rpm \
- && zypper -n clean \
+ && wget https://ftp.gwdg.de/pub/opensuse/repositories/home%3A/Herbster0815/openSUSE_Leap_15.1/noarch/automake-1.16.1-lp151.92.1.noarch.rpm \
```

Comment:
I'm a little worried about installing a package from Leap 15.1 on Leap 42.3. Can we not find a build for Leap 42.3?

Reply:
This seems to work, though, so let's keep it.